__tests__/app.test.js

```diff
@@ -1,41 +1,34 @@
 const pool = require('../lib/utils/pool');
 const setup = require('../data/setup');
-const request = require('supertest');
+//const request = require('supertest');
 const app = require('../lib/app');
-
-describe('users', () => {
-  beforeEach(() => {
-    return setup(pool);
-  });
-
+const request = require('supertest');
+//const UserService = require('../lib/services/UserService');
 
 const sampleUser = {
-  firstName: 'Test',
-  lastName: 'Testerson;',
+  first_name: 'Test',
```

Comment:
you want to keep it ```firstName``` as ```camelCase``` is convention for multiple word variable names in JavaScript. ```snake_case``` is appropriate for database columns. if you did this to fix something it should probably be fixed another way